src/main/java/athena/Ui.java

```diff
@@ -0,0 +1,2 @@
+package athena;public class Ui {
```

Comment:
Make sure to have this on two seperate lines

Reply:
Hmm not sure why it shows up like this on github but I definitely had it on two separate lines in my IDE